setup_docs.py

```diff
@@ -470,7 +471,10 @@ def write_man_header(self, write, title, description):
         self.write_heading(write, description, double_sided=True)
         # man page metadata
         write(":Author: The Borg Collective")
-        write(":Date:", datetime.utcnow().date().isoformat())
+        write(
+            ":Date:",
+            datetime.utcfromtimestamp(int(os.environ.get("SOURCE_DATE_EPOCH", time.time()))).date().isoformat(),
```

Comment:
making this reproducible is fine, but why was the default code (when the env var is not set) changed?

Reply:
I can only second guess here, but I think the idea was to avoid branching. `datetime.utcnow()` and `datetime.utcfromtimestamp(int(time.time()))` shouldn't be that different.

I'll summon @vagrantc for a more detailed answer.

Reply:
hmm, ok, result seems to be the same.

Reply:
@helmutg no need, i first had the wrong idea about what's in that env var and got the bracketing wrong...